fdbserver/RestoreMaster.actor.cpp

```diff
@@ -226,15 +226,21 @@ ACTOR Future<Void> startProcessRestoreRequests(Reference<RestoreMasterData> self
 
 ACTOR static Future<Version> processRestoreRequest(Reference<RestoreMasterData> self, Database cx,
                                                    RestoreRequest request) {
-	state std::vector<RestoreFileFR> files;
+	state std::vector<RestoreFileFR> rangeFiles;
+	state std::vector<RestoreFileFR> logFiles;
 	state std::vector<RestoreFileFR> allFiles;
 	state std::map<Version, VersionBatch>::iterator versionBatch = self->versionBatches.begin();
 
 	self->initBackupContainer(request.url);
 
 	// Get all backup files' description and save them to files
-	wait(collectBackupFiles(self->bc, &files, cx, request));
-	self->buildVersionBatches(files, &self->versionBatches); // Divide files into version batches
+	wait(collectBackupFiles(self->bc, &rangeFiles, &logFiles, cx, request));
+
+	std::sort(rangeFiles.begin(), rangeFiles.end());
+	std::sort(logFiles.begin(), logFiles.end());
+
+	self->buildVersionBatches(rangeFiles, logFiles, &self->versionBatches); // Divide files into version batches
+	self->dumpVersionBatches(self->versionBatches);
```

Comment:
I tend to view this as an non-debugging function. It provides a global view of what files will be processed for which version range in each version batch.
when it's in production, this can still be useful information. 
It is only printed out once at the beginning of the restore request.

Reply:
Ok that seems fine, though be aware that if the number of these files is large, you may end up getting suppressed automatically.

Reply:
It should be fine to suppress SevInfo. As long as SevError won't be suppressed.